src/ast/passes/semantic_analyser.cpp

```diff
@@ -1178,6 +1224,33 @@ void SemanticAnalyser::visit(Call &call)
 
     call.type = CreateUInt64();
   }
+  else if (call.func == "offsetof")
+  {
+    if (check_nargs(call, 2))
+    {
+      // Handle Identifier only, see visit(Call) of offsetof()
```

Comment:
Comment is a bit confusing. Suggestion:
```suggestion
      // Handle Identifier only, see earlier in this function where we accept()  call arguments
```

Reply:
Fix it, thanks.